merlin-server/src/main/java/gov/nasa/jpl/aerie/merlin/server/services/LocalMissionModelService.java

```diff
@@ -27,7 +29,7 @@
  * connected mission model repository.
  */
 public final class LocalMissionModelService implements MissionModelService {
-  private static final Logger log = Logger.getLogger(LocalMissionModelService.class.getName());
+  private static final Logger log = LoggerFactory.getLogger(LocalMissionModelService.class);
```

Comment:
Thanks for making this change! Do we also want to update these instances as well?
```
develop:aerie$ ag Logger.getLogger
merlin-server/src/main/java/gov/nasa/jpl/aerie/merlin/server/AerieAppDriver.java
36:  private static final Logger log = Logger.getLogger(AerieAppDriver.class.getName());

merlin-server/src/main/java/gov/nasa/jpl/aerie/merlin/server/services/LocalMissionModelService.java
30:  private static final Logger log = Logger.getLogger(LocalMissionModelService.class.getName());

scheduler-server/src/main/java/gov/nasa/jpl/aerie/scheduler/server/http/SchedulerBindings.java
43:  private static final Logger log = Logger.getLogger(SchedulerBindings.class.getName());
```

Reply:
Looks like you ran that on develop. I removed the instance in AerieAppDriver because it was never used, and I'll get to the instances in scheduler in AERIE-1734. Thanks!

Reply:
Ohh gotcha, perfect!